starboard/linux/shared/BUILD.gn

```diff
@@ -271,6 +270,9 @@ static_library("starboard_platform_sources") {
     "//starboard/shared/stub/thread_create_priority.cc",
     "//starboard/shared/stub/window_get_diagonal_size_in_inches.cc",
   ]
+  if (enable_evergreen_code) {
```

Comment:
Why would this need to be excluded? Are we not defining the SB_IS macro?

Reply:
it depends on evergreen symbols, ran into an error while building earlier.
For ex:
EvergreenInfo evergreen_info;

Can re-enable in a follow up alongwith other evergreen related code

Reply:
Could you make a task on the taskflow for this and assign it to me please?